### PR TITLE
[VBLOCKS-2476] feat(js): use iso timestamps

### PR DESCRIPTION
**Pull request by mhuynh5757**
## Submission Checklist

 - [x] Updated the `CHANGELOG.md` to reflect any **feature**, **bug fixes**, or **known issues** made in the source code
 - [x] Tested code changes and observed expected behavior in the example app
 - [x] Performed a visual inspection of the `Files changed` tab prior to submitting the pull request for review to ensure proper usage of the style guide

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Description

This PR changes the public API to reflect the native layer changes of using ISO timestamps instead of milliseconds from epoch for the initial connected time.

## Breakdown

- Parse ISO timestamps using the built-in Date object.
- Change public API to reflect the new type.

## Validation

- Unit tests.
- Manual testing on Android and iOS.

## Additional Notes

N/A

### CODE REVIEW

**Review by bobiechen-twilio — APPROVED** [review]
iOS part looks good. assuming verified with the test app

**Review by afalls-twilio — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+1.0.0-beta.5 (In Progress)
+==========================
 
 Twilio Voice React Native SDK has now reached milestone `beta.5`. Included in this version are the following.
 
 ## Changes
 
 - Fixed and improved the docstrings for the `Voice` and `Call` listeners. The descriptions of the events and listeners should now point to the correct docstrings.
+- The API for `call.getInitialConnectedTimestamp()` has now changed.
+  Please see the API documentation for details.
```
Comment (charliesantos) [inline CHANGELOG.md:10]: Add link to the doc for this api

**Review by charliesantos — APPROVED** [review]
+1. Please address my minor comment before merging.

**Review by charliesantos — APPROVED** [review]

**Pull request merged**